locale/English.json

```diff
@@ -46,8 +46,9 @@
       "btnRisky": "(Dangerous!)",
       "qrContentCopied": "Copied QR Content",
       "translatedText": "Translated Text",
-      "feedbackSuccess":"You have Successfully Submitted Feedback.Thank You",
-      "feedbackFailure": "Something went wrong...Please try again "
+      "feedbackSuccess": "You have Successfully Submitted Feedback.Thank You",
+      "feedbackFailure": "Something went wrong...Please try again ",
```

Comment:
"Something went wrong... Please try again."

Btw, I am not a fan of telling "something", that doesn't give any hint, and for users it feels like we are clueless. I would say, "Failed to submit feedback at this time. Please try again in sometime" may sound better. 

@sankarshanmukhopadhyay @mankuthimma what do you think?

Reply:
Perhaps something like "Oops! Your feedback didn't get through ... Please try again!" might keep the tone light and engaging.